### ISSUE TITLE
[Programmatic Access – Make Code - Zoom In/ Zoom Out] Incorrect role is defined for 'Undo/Redo/Zoom In/Zoom Out' button control present under 'Editor screen'.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3598]
**User Experience:**
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is a link or a button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces menu item while navigating in scan mode and hence it will not be a good usable experience for the users.

**Note:** User credentials should NOT be included in the bug.

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL 
2.  Home page will open.
3. Navigate "New Project" option and select it.
4.  Editor screen will open.
5.  Navigate to "Undo/Redo/Zoom In/Zoom Out" button and select it.
6.  Verify user able to perform all action using the controls.
7.  Verify the issue incorrect role 'menu Item' is define for Undo/Redo/Zoom In/Zoom Out control or not.

**Actual Result:**
Incorrect role is defined for 'Undo/Redo/Zoom In/Zoom Out' button control present under 'Editor screen.

**Expected Result:**
Correct role should be define as 'Button' instead of menu item for 'Undo/Redo/Zoom In/Zoom Out' button control present under 'Editor screen' pop-up.  Also screen reader announce information for the zoom in/out percentage.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**WCAG Reference:**
[WCAG 4.1.2 – Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)

**Test Environments:**
Version: 2004 (OS Build: 19041.572)
Browser: Edge chromium DEV Build (Version 88.0.685.3), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)  


<img width="961" alt="WCAG_4 1 2_Incorrect role is defined for 'Undo,Redo,Zoom In,Zoom Out' button control present under 'Editor screen'" src="https://user-images.githubusercontent.com/72871089/98238426-5e207500-1f8c-11eb-8142-db0961d99213.png">

**Comment by jwunderl** [comment]
These are part of the bottom menu so having them listed as menu items seems correct
